### DISCUSSION TITLE
MIMIC IV Note

### DISCUSSION THREAD

**Discussion by fantamanatee** [MIT-LCP/mimic-code#1490] [MIMIC-IV]
In the [Load MIMIC-IV into a PostgreSQL database tutorial](https://github.com/MIT-LCP/mimic-code/tree/main/mimic-iv/buildmimic/postgres#load-mimic-iv-into-a-postgresql-database), I noticed note events are not included. Since MIMIC IV note was published on Jan 6, is it possible to include MIMIC IV note data in a local postgresql database (while also following the postgresqk tutorial)?

**Comment by alistairewj** [comment]
Ah yes, TODO! This actually makes more sense as an issue.
